recipes/multi-hauser/README.md

```diff
@@ -0,0 +1,72 @@
+# Multi-Hauser with Docker
+
+If your FullStory account has multiple associated accounts, this recipe can be used as a template to
+start a docker container that exports the data for each "org" and uploads it to BigQuery.
+While this is intended to work with Google Cloud, it can easily be adapted to work with
+AWS or any databases supported in the future.
+
+## How it works
+This recipe works by creating a docker container that uses [supervisord] to run multiple `hauser`
```

Comment:
Is the link to supervisord perhaps missing?  I see the brackets so I figure it was intended to be a link.

Reply:
It's not missing (see the references at the bottom), but it is broken! Looks like I need to add `http://` to make sure it's not a relative link.